### ISSUE TITLE
Localisation - Fix font issue for Greek

### ISSUE THREAD

**Issue by doheekim93** [FightPandemics/FightPandemics#2181]
As mentioned in #2166, we observed some font inconsistencies for Greek (different fonts seen on `About Us` and `Help Board`).

Example from `About Us`:
<img width="885" alt="Screen Shot 2021-01-25 at 11 13 44 AM" src="https://user-images.githubusercontent.com/66695502/105740118-7acea800-5efe-11eb-8718-ee0778e42d54.png">

We had a similar problem in the past because we didn't have all the correct weights of the font downloaded. Please refer to this [PR](https://github.com/FightPandemics/FightPandemics/pull/1868) to see if we can solve the issue in a similar way, thanks!

**Comment by doheekim93** [comment]
Hey @emanuelquerty, did you want to work on this? We merged the other PR, so I opened this new ticket.

**Comment by emanuelquerty** [comment]
Hey @doheekim93 yeah I did. Thanks for letting me know.

I fixed the problem and it was for a different reason than what we had in this [PR](https://github.com/FightPandemics/FightPandemics/pull/1868). It was because the css rules for font-family was missing the default fallback font (i.e.`font-family: "Poppins";` vs `font-family: "Poppins", sans-serif;`).  It appears that both Poppins and Work Sans are not fully supported in all systems/browsers and both are actually falling back to some version of Sans-Serif in some cases. Since there was no fallback in CSS, hence the problem. 

 I will open the PR now and push the fix.
